lib/mongo/bulk_write.rb

```diff
@@ -39,7 +39,7 @@ class BulkWrite
                    :database,
                    :cluster,
                    :write_with_retry,
-                   :legacy_write_with_retry,
+                   :nro_write_with_retry,
```

Comment:
Out of curiosity, what does "nro" stand for?

Reply:
"Non-retryable operation".